components/Button.tsx

```diff
@@ -4,6 +4,7 @@ interface Props {
   size?: "small";
   icon?: React.ReactNode;
   onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  href?: string;
```

Comment:
Hvorfor trenger vi denne? Når man gjøre i det onClick. onClick={() => router.push("/")}

Reply:
Hvis vi kun har onClick={...} så kan man ikke håndtere den som en link. Kan feks ikke åpne linken i ny fane, kan ikke hovre over linken og se hvor den tar deg hen.

Reply:
> Hvis vi kun har onClick={...} så kan man ikke håndtere den som en link. Kan feks ikke åpne linken i ny fane, kan ikke hovre over linken og se hvor den tar deg hen.

Skjønner, da burde vi vel endre alle steder vi har brukt onClick={} for navigasjon?

Reply:
Ja, tenkte å gjøre det etterhvert